lib/hanami/model/sql/types.rb

```diff
@@ -34,26 +34,26 @@ module Schema
           # @since 0.7.0
           # @api private
           MAPPING = {
-            Types::String.with(meta: {})            => Schema::String,
-            Types::Int.with(meta: {})               => Schema::Int,
-            Types::Float.with(meta: {})             => Schema::Float,
-            Types::Decimal.with(meta: {})           => Schema::Decimal,
-            Types::Bool.with(meta: {})              => Schema::Bool,
-            Types::Date.with(meta: {})              => Schema::Date,
-            Types::DateTime.with(meta: {})          => Schema::DateTime,
-            Types::Time.with(meta: {})              => Schema::Time,
-            Types::Array.with(meta: {})             => Schema::Array,
-            Types::Hash.with(meta: {})              => Schema::Hash,
-            Types::String.optional.with(meta: {})   => Schema::String,
-            Types::Int.optional.with(meta: {})      => Schema::Int,
-            Types::Float.optional.with(meta: {})    => Schema::Float,
-            Types::Decimal.optional.with(meta: {})  => Schema::Decimal,
-            Types::Bool.optional.with(meta: {})     => Schema::Bool,
-            Types::Date.optional.with(meta: {})     => Schema::Date,
-            Types::DateTime.optional.with(meta: {}) => Schema::DateTime,
-            Types::Time.optional.with(meta: {})     => Schema::Time,
-            Types::Array.optional.with(meta: {})    => Schema::Array,
-            Types::Hash.optional.with(meta: {})     => Schema::Hash
+            Types::String.pristine            => Schema::String,
+            Types::Int.pristine               => Schema::Int,
+            Types::Float.pristine             => Schema::Float,
+            Types::Decimal.pristine           => Schema::Decimal,
+            Types::Bool.pristine              => Schema::Bool,
+            Types::Date.pristine              => Schema::Date,
+            Types::DateTime.pristine          => Schema::DateTime,
+            Types::Time.pristine              => Schema::Time,
+            Types::Array.pristine             => Schema::Array,
+            Types::Hash.pristine              => Schema::Hash,
+            Types::String.optional.pristine   => Schema::String,
```

Comment:
@flash-gordon What is "this", specifically? 😉  `#pristine`? `#optional` types? The whole mapping?